pkg/kubectl/cmd/util/shortcut_restmapper.go

```diff
@@ -112,6 +112,8 @@ var UserResources = []schema.GroupResource{
 	{Group: "batch", Resource: "jobs"},
 	{Group: "extensions", Resource: "deployments"},
 	{Group: "extensions", Resource: "replicasets"},
+	{Group: "extensions", Resource: "ingresses"},
+	{Group: "extensions", Resource: "daemonsets"},
```

Comment:
Good to know, but I think that we may need to update the help info for `kubectl get all` a bit to reflect this.

Also I think that we may need to expose more resources here which maybe required by end user for some troubleshooting, such as daemonset, ingress etc.

Comments? @liggitt